packages/core/src/sequence.ts

```diff
@@ -35,11 +72,21 @@ export class Sequence {
       this.sendError(res, req, err);
     }
   }
-
+   /**
+   * Sends API result in HTTP response
+   *
+   * @param res HTTP server response to write the result of method invocation
+   * @param result Result from method invocation
```

Comment:
Should this be `@param response` since in code, param is named `response` not `res`?